projects/1.18/assets/chipped/chipped/lang/zh_cn.json

```
@@ -1,2510 +1,7786 @@
+    "block.chipped.glowstone_18": "精致荧石灯笼",
+    "block.chipped.glowstone_19": "精饰荧石灯笼",
+    "block.chipped.glowstone_20": "下界荧石",
+
+    "block.chipped.gold_block_1": "叠板金块",
+    "block.chipped.gold_block_2": "方格金块",
+    "block.chipped.gold_block_3": "压纹金块",
+    "block.chipped.gold_block_4": "压制金块",
+    "block.chipped.gold_block_5": "强化金块",
+    "block.chipped.gold_block_6": "叠层金块",
+    "block.chipped.gold_block_7": "层纹金块",
+    "block.chipped.gold_block_8": "窗格金块",
+    "block.chipped.gold_block_9": "凿制金块",
+    "block.chipped.gold_block_10": "雕纹金块",
+    "block.chipped.gold_block_11": "宽板金块",
+    "block.chipped.gold_block_12": "古式金块",
+
+    "block.chipped.granite_1": "磨制花岗岩",
+	"block.chipped.granite_2": "微型砖沿花岗岩",
+	"block.chipped.granite_3": "雕纹花岗岩",
+	"block.chipped.granite_4": "环纹花岗岩",
+	"block.chipped.granite_5": "不悦徽花岗岩浮雕",
+	"block.chipped.granite_6": "嵌花花岗岩",
+	"block.chipped.granite_7": "粗糙花岗岩",
+	"block.chipped.granite_8": "铺砌花岗岩砖",
+	"block.chipped.granite_9": "柱式花岗岩砖",
+	"block.chipped.granite_10": "密层花岗岩砖",
+	"block.chipped.granite_11": "蜘蛛徽花岗岩浮雕",
+	"block.chipped.granite_12": "惊讶徽花岗岩浮雕",
+	"block.chipped.granite_13": "砖沿花岗岩",
+	"block.chipped.granite_14": "错排花岗岩砖",
+	"block.chipped.granite_15": "花岗岩砖",
+	"block.chipped.granite_16": "花岗岩小型瓦",
+	"block.chipped.granite_17": "悲伤徽花岗岩浮雕",
+	"block.chipped.granite_18": "愤怒徽花岗岩浮雕",
+	"block.chipped.granite_19": "空白式花岗岩浮雕",
+	"block.chipped.granite_20": "欢愉徽花岗岩浮雕",
+	"block.chipped.granite_21": "圆石纹花岗岩", 
+	"block.chipped.granite_22": "刻沿花岗岩砖",
+	"block.chipped.granite_23": "苦力怕徽花岗岩浮雕",
+	"block.chipped.granite_24": "风蚀花岗岩",
+  	"block.chipped.granite_25": "蚀刻花岗岩砖",
+  	"block.chipped.granite_26": "平整花岗岩瓦",
+  	"block.chipped.granite_27": "厚沿嵌花花岗岩",
+  	"block.chipped.granite_28": "纂刻花岗岩",
+  	"block.chipped.granite_29": "带框花岗岩",
+  	"block.chipped.granite_30": "磁石纹花岗岩",
+  	"block.chipped.granite_31": "裂纹乱砌花岗岩砖",
+  	"block.chipped.granite_32": "复纹花岗岩瓦",
+  	"block.chipped.granite_33": "裂纹花岗岩砖",
+  	"block.chipped.granite_34": "裂纹平整花岗岩瓦",
+  	"block.chipped.granite_35": "海晶纹花岗岩",
+  	"block.chipped.granite_36": "未刻花岗岩浮雕",
+  	"block.chipped.granite_37": "圆角花岗岩砖",
+  	"block.chipped.granite_38": "符文花岗岩浮雕",
+  	"block.chipped.granite_39": "切制花岗岩柱石",
+  	"block.chipped.granite_40": "沙纹花岗岩",
+  	"block.chipped.granite_41": "花岗岩小型砖",
+  	"block.chipped.granite_42": "平滑花岗岩柱石",
+  	"block.chipped.granite_43": "平滑嵌花花岗岩",
+  	"block.chipped.granite_44": "平滑回纹花岗岩",
+  	"block.chipped.granite_45": "平滑双重嵌花花岗岩",
+  	"block.chipped.granite_46": "棋盘格花岗岩瓦",
+  	"block.chipped.granite_47": "螺旋纹花岗岩",
+  	"block.chipped.granite_48": "叠层花岗岩砖",
+  	"block.chipped.granite_49": "失望徽花岗岩浮雕",
+  	"block.chipped.granite_50": "方纹花岗岩",
+  	"block.chipped.granite_51": "方纹花岗岩柱石",
+  	"block.chipped.granite_52": "方纹带框花岗岩",
+  	"block.chipped.granite_53": "花岗岩微型砖",
+  	"block.chipped.granite_54": "刻制花岗岩",
+  	"block.chipped.granite_55": "细层花岗岩砖",
+  	"block.chipped.granite_56": "普通花岗岩柱",
+  	"block.chipped.gra
```

```suggestion
  	"block.chipped.lodestone_28": "篆刻磁石",
```
`纂`是`编`的意思
看原文应该是篆刻的`篆`(zhuan)。